src/python/nimbusml/base_predictor.py

```diff
@@ -55,6 +57,13 @@ def fit(self, X, y=None, **params):
         set_shape(self, X)
         return self
 
+    @property
+    def has_implicit_predictors(self):
+        """
+        Tells if the predictor depends on other predictors.
+        """
+        return self._has_implicit_predictors
```

Comment:
>return self._has_implicit_predictors [](start = 8, length = 36)

Didnt get it. What is the purpose of this?

Reply:
This method was added to the base class of predictors so that the code in Pipeline can determine which predictors have implicit predictors (the voting ensembles) and thus generate the graph in a different way.

This was done to avoid having `if classname == "VotingRegressor" line in Pipeline.

---
In reply to: [361724371](https://github.com/microsoft/NimbusML/pull/390#discussion_r361724371) [](ancestors = 361724371)